src/Package/Impl/ProjectSystem/RDebugLaunchProvider.cs

```diff
@@ -45,6 +46,7 @@ public RDebugLaunchProvider(ConfiguredProject configuredProject, IRInteractiveWo
         internal IFileSystem FileSystem { get; set; } = new FileSystem();
 
         private IRSession Session => _interactiveWorkflow.RSession;
+        private IRCallbacks Callbacks => (IRCallbacks)_interactiveWorkflow.RSession;
```

Comment:
It looks like this is to output to the REPL. I believe this should go through `IRInteractiveWorkflow.ActiveWindow.InteractiveWindow.Write...`?